packages/app/src/components/ErrorBoundaries/FatalErrorBoundary.js

```diff
@@ -0,0 +1,179 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@dhis2/d2-i18n';
+import { withStyles } from '@material-ui/core/styles';
+import InfoIcon from '@material-ui/icons/InfoOutlined';
+
+const bgColor = '#F4F6F8',
```

Comment:
we have these colors defined in colors.js "snow" and  "greyDark". (I know, great names...)

Reply:
Yeah, they're also in the theme but I wanted to keep dependencies to a minimum so I decided to hard-code them.  I'll import them directly from the mui3.there object (not referencing the mui3 theme) since that's becoming the canonical source of truth and the tests should catch any issues with dereferencing those variables.

Reply:
Done